packages/testsuite/cypress/e2e/elytron/test-configuration-subsystem-elytron-others-settings.cy.ts

```diff
@@ -0,0 +1,69 @@
+describe("TESTS: Configuration => Subsystem => Security => Settings => Global Settings", () => {
+  let managementEndpoint: string;
+
+  const address = ["subsystem", "elytron", "security-domain", "ApplicationDomain"];
+
+  const navigation = {
+    sslButton: "#ssl-item",
+    securityDomainButton: "#elytron-security-domain-item",
+    securityDomainRealmsButton: "#hal-uid-5",
+  };
+
+  const realmsForm = {
+    id: "elytron-security-domain-realms-add",
+    addButton: "elytron-security-domain-realms-table",
+    principalTransformer: {
+      id: "#elytron-security-domain-realms-add-editing > div:nth-child(2)",
```

Comment:
there's lot of repetions of `#elytron-security-domain-realms-add-editing` and `nth-child` should be avoided for clarity if possible. Consider change like this: https://github.com/hal/berg/blob/c79dbe621c98ce5a43ee0bc595e25f37650112ff/packages/testsuite/cypress/e2e/ejb/test-configuration-subsystem-ejb-container-remoting-profile.cy.ts#L160-L163